src/test/java/Homework19.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class Homework19 extends BaseTest {
+    //1. Create a new file Homework19.java with @Test annotation in IntelliJ IDEA.
+    //2. Create a test case deletePlaylist() using @Test annotations.
+    //3. Use the helper/reusable methods we created.
+    //4. Use @Parameters for passing baseUrl from the TestNG config file to the tests.
+    //5. Navigate to "https://bbb.testpro.io/".
+    //6. Log in with your credentials.
+    //7. Choose a playlist.
+    //8. Delete the playlist
+    //9. Validate that the playlist is deleted.
+    //10. Create a new branch and commit your changes.
+    //11. Push your code to a remote repository.
+    //12. Create a pull request.
+    @Test
+    @Parameters({"baseURL"})
```

Comment:
The @Parameters method is already defined in BaseTest class (launchBrowser method)